apps/fabric-website/webpack.config.js

```diff
@@ -31,6 +32,22 @@ module.exports = function(env) {
         chunkFilename: `${entryPointName}-${version}-[name]-${now}${minFileNamePart}.js`
       },
 
+      module: {
```

Comment:
I'm concerned we may need this same set of rules and plugins (and probably the monaco webpack dep) in anything depending on example-app-base. If so, it should probably go in webpack-resources.js. @kenotron any thoughts on this?